tests/utils/test_gravatar.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+from libgravatar import Gravatar
+
+
+class TestGravatar(unittest.TestCase):
+
+    def test_works(self):
+        valid_photo_url = "http://www.gravatar.com/avatar/b642b4217b34b1e8d3bd915fc65c4452?size=512"
+
```

Comment:
Do we need this line break?

Reply:
All those line breaks are recommended by PEP8.